spec/Section 5 -- Validation.md

```diff
@@ -568,7 +568,7 @@ fragment conflictingDifferingResponses on Pet {
 
 **Explanatory Text**
 
-Field selections on scalars are never allowed: scalars
+Field selections on scalars or enums are never allowed: scalars and enums
```

Comment:
nit: maybe instead of `: scalars and enums` write `because they are the leaf nodes of any GraphQL query.`